.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: CI
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - '.gitignore'
+      - 'LICENSE'
+      - '**.md'
+
+jobs:
+  test:
+    name: Test Runner
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Build Docker Image
+        run: docker build -f Dockerfile -t c-test-runner .
```

Comment:
Could you change the tag name from c-test-runner to x86-64-assembly-test-runner?

Reply:
Oops; missed that. Addressing that now.